.github/workflows/test.yaml

```
@@ -18,29 +18,53 @@ jobs:
       - uses: pre-commit/action@v2.0.0
 
   pytest:
-    name: "Run pytests (FIXME: No tests)"
+    name: "Run tests"
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
         include:
           - python-version: 3.6
+            jupyterhub-version: 1.3.*
           - python-version: 3.7
           - python-version: 3.8
+            jupyterhub-version: 1.*
           - python-version: 3.9
+            jupyterhub-version: 2.0.*
```

```suggestion
          - python-version: 3.6
            jupyterhub-version: 1.3.*
          - python-version: 3.7
            jupyterhub-version: 1.*
          - python-version: 3.8
            jupyterhub-version: 2.0.*
          - python-version: 3.9
            jupyterhub-version: 2.*
```

What do you think about this? I like that it becomes clear that we will test against the latest version of jupyterhub with a quick inspection - at least until we release JupyterHub 3.